test/format/wts.c

```diff
@@ -336,6 +336,17 @@ configure_chunkcache(char **p, size_t max)
     }
 }
 
+/*
+ * configure_prefetch --
+ *     Configure prefetch cache settings for opening a connection.
```

Comment:
I understand the confusion and I think it should be indeed enabled in the config so we have the following:

- Enable prefetch in the config
- If verify is enabled, and prefetch has been enabled in the config, randomly enable it for verify. But now I am wondering, should randomly enable prefetch for any session @mm-ng ?

Reply:
This means that inside `wt_wrap_open_session`, we could do:

```
if (GV(OPS_VERIFY)) && some_random_stuff)
  config += prefetch=(enabled=true)
```

This would be great because it would potentially impact **any read/write**  and not only `verify`. What do you think @kommiharibabu @mm-ng?

Reply:
@etienneptl I think it would be nice to have functionality to enable pre-fetching in read scenarios that are not necessarily from verify operations, e.g. linear scans (I don't think it should really affect writes). Perhaps we could generate these different types of testing for maximum coverage: enable pre-fetch for both verify and reads, enable pre-fetch for verify only, enable pre-fetch for reads but not verify explicitly.

Reply:
Would that be an issue to have prefetch enable at all times in a test/format run? Each session could have prefetch, whether it does reads or writes.

Reply:
@etienneptl I was thinking it could be enabled only for a certain percentage of times, it definitely doesn't have to be for all times/all sessions.

Reply:
The more variability the better.

Reply:
I have updated the code and added some randomness for various operations, let me know what you think.